miio/airconditioningcompanion.py

```diff
@@ -310,33 +310,29 @@ def learn_stop(self, slot: int=STORAGE_SLOT_ID):
         return self.send("end_ir_learn", [slot])
 
     @command(
-        click.argument("model", type=str),
-        click.argument("code", type=str),
+        click.argument("model", type=bytes),
+        click.argument("code", type=bytes),
         default_output=format_output("Sending the supplied infrared command")
     )
-    def send_ir_code(self, model: str, code: str, slot: int=0):
+    def send_ir_code(self, model: bytes, code: bytes, slot: int=0):
```

Comment:
I've been thinking about keeping strings here as args but converting them to `bytearray`/`bytes` right away. Having `bytes` as arg type for command line is probably not very good idea :). Maybe there should be two methods? Main one taking `bytes` like this one, but the `@command` should be applied to a wrapper method, which takes hex strings, converts them and calls the `bytes` one.

Reply:
Optionally (maybe even better solution) would be to add converting functions to click arguments definition to convert hex strings from command line to bytes.